translations/messages.pt_BR.xliff

```diff
@@ -837,6 +928,22 @@
                 <source>entryState.stopped</source>
                 <target>Parado</target>
             </trans-unit>
+
+
+            <trans-unit id="export.clear_all">
+                <source>export.clear_all</source>
+                <target>Marcar todos os registros mostrados como aberto?</target>
+            </trans-unit>
+            <trans-unit id="export.mark_all">
+                <source>export.mark_all</source>
+                <target>Mark all shown records as cleared?</target>
```

Comment:
this line needs to be removed, probably the build will work then

Reply:
My mistake, fixed now